### PR TITLE
moved contacts formats to ContactExchangeFormats.kt

### PR DESCRIPTION
**Pull request by JoGir**

### CODE REVIEW

File: core/src/test/java/de/qabel/core/contacts/ContactExchangeFormatsTest.kt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        val contact2KeyPair = QblECKeyPair()
+        contact2 = Contact("Contact2", dropURLs, contact2KeyPair.pub)
+
+        contacts = Contacts(identity)
+        contacts.put(contact1)
+        contacts.put(contact2)
+    }
+
+    private fun assertContactEquals(contact1: Contact, contact2: Contact) {
+        assertThat(contact1.alias, Is.`is`(contact2.alias))
```
Comment (audax) [inline core/src/test/java/de/qabel/core/contacts/ContactExchangeFormatsTest.kt]: move the Is.is stuff into another method, it's ugly :D

**Comment by JoGir** [comment]
... java.lang.IllegalStateException: Illegal response from accounting server

**Comment by JoGir** [comment]
resolves https://github.com/Qabel/qabel-android/issues/269

File: core/src/test/java/de/qabel/core/contacts/ContactExchangeFormatsTest.kt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        contacts.put(contact1)
+        contacts.put(contact2)
+    }
+
+    private fun assertContactEquals(contact1: Contact, contact2: Contact) {
+        assertThat("Alias doesnt match", contact1.alias.equals(contact2.alias))
+        assertThat("DropUrls dont match", contact1.dropUrls.equals(contact2.dropUrls))
+        assertThat("PubKey doesnt match", contact1.ecPublicKey.readableKeyIdentifier.
+            equals(contact2.ecPublicKey.readableKeyIdentifier))
+        assertThat("Phone doesnt match", contact1.phone?.equals(contact2.phone) ?: contact2.phone == null)
```
Comment (audax) [inline core/src/test/java/de/qabel/core/contacts/ContactExchangeFormatsTest.kt]: == is equals() and === is the referential identity check (== in java)

**Pull request merged**